src/bentoml/_internal/env_manager/manager.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+import os
+import typing as t
+import logging
+import subprocess
+from tempfile import NamedTemporaryFile
+from distutils.spawn import find_executable as which
+
+import fs
+import attrs
+from rich.status import Status
```

Comment:
We are not depending on rich as a dependency, I suggest not doing this as rich is pretty slow.

Reply:
but we are depending on rich already right? the idea was to put in a spinner but since we already have rich I thought  we will use the one already there